client/app/templates/template-explorer.component.js

```diff
@@ -131,18 +142,26 @@ function ComponentController(ListView, TemplatesService, EventNotifications, $st
       name: __('Edit'),
       title: __('Edit Template'),
       actionFn: handleEdit,
-    }];
+    },
+    {
+      name: __('Delete'),
+      title: __('Delete Template'),
+      actionFn: handleDelete,
+    },
+    ];
 
     return menuActions;
   }
   function listActionDisable(config, items) {
     const menuCreate = 0;
     const menuEdit = 1;
+    const menuDelete = 2;
 
     switch (config.actionName) {
       case 'configuration':
         config.actions[menuCreate].isDisabled = items.length > 0;
```

Comment:
so we're disabling create option if anything in the list is selected? Not sure I understand... this might be a broader conversation collectively we need to have, determining what actions should be disabled when... i have a feeling we'll find many inconsistencies throughout the app :P